scripts/data_contamination/requirements.txt

```diff
@@ -0,0 +1,43 @@
+apache-beam==2.47.0
```

Comment:
Do we need everything? I think we only need `apache-beam`, `bitarray` `nltk`, and `tqdm`? Or is this the entire frozen dependency tree?

Also I wonder if the inclusion of `beam` (vs `apache-beam`) is a typo.

Reply:
Thanks for the catch, `beam` is a typo and removed. All of the dependencies are recursive dependencies of those you mentioned (besides `pyarrow` which I introduced to read parquet files). I've pruned the dependencies list to these core dependencies for ease of reading